src/main/java/seedu/address/logic/commands/SchemeCommand.java

```diff
@@ -62,6 +62,7 @@ public CommandResult execute(Model model) throws CommandException {
         if (result.toString().equals("")) {
             result = new StringBuilder("No schemes available for this family.");
         }
+        result.insert(0, "Schemes available for family: " + targetFamily.getName() + "\n");
```

Comment:
Same thing here

Reply:
have updated references to family to person for consistency, thanks for pointing it out!